Armadillo/src/main/java/com/scribd/armadillo/download/DownloadEngine.kt

```diff
@@ -70,8 +70,7 @@ internal class ExoplayerDownloadEngine @Inject constructor(private val context:
         })
     }
 
-    override fun removeDownload(audiobook: AudioPlayable) =
-        DownloadService.sendRemoveDownload(context, downloadService, audiobook.request.url, true)
```

Comment:
Do we want to remove `sendRemoveDownload` entirely?

Reply:
I believe so